### ISSUE TITLE
[BUG]: All playlists in sidebar show "Can't load content"

### ISSUE THREAD

**Issue by Lynquid** [yuliskov/SmartTube#4099]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

24.65 Beta (I think)

### Device Type

Smart TV/Box

### Affected Android

Fire OS 7.6.8.5 (PS7685/4486)

### Steps to reproduce the bug

Go to a Playlist in the sidebar

### Actual behavior

All of them show "Can't load content", the playlists section works tho. It seems like this is a bug that only I have.

Here's a video that shows it.
https://github.com/user-attachments/assets/503380f5-e523-407f-b023-b5336de8522d

**Comment by Fat-Zilla** [comment]
I get this with Alphabetical (fast previews) on the firestick. I can change to default or the broken Alphabetical and get channels to atleast show.

**Comment by rvk01** [comment]
I don't think this happened with a version-change but because of a change at Youtube. I have Watch later in the sidebar and it gives me the "can't load content". I know for sure it worked a few days ago with the same version (24.64 on both nokia streamingbox and konstakang android tv on rpi4). I can view the Watch later under the Playlists but it read easier in the sidebar. The other menu items work. Other items (like music playlist of news channel) added to the sidebar work. Just the ones added from Playlists don't.

**Comment by jdgremsjr** [comment]
This is now happening to me as well. It was only a temporary problem until recently now all 5 of my devices are doing it all the time.

**Comment by rvk01** [comment]
Still not fixed in 24.71. Now it doesn't even give you an error. Just a blank page.

**Comment by rvk01** [comment]
This is fixed in 24.89

**Comment by Fat-Zilla** [comment]
> This is fixed in 24.89

Its not fully fixed in the latest live release. Alphabetical still is not working correctly. Its not in alphabetical order. It does display content at least.
